.vscode/settings.json

```diff
@@ -1,3 +1,10 @@
 {
-  "debug.node.autoAttach": "on"
+  "debug.node.autoAttach": "on",
```

Comment:
Was this file changed on purpose? If so, can you explain the changes? Otherwise everything else LGTM!

Reply:
No! I'll remove it again. Glad to hear it's otherwise fine!